app/src/main/java/com/example/coronadiagnosticapp/data/providers/SharedProviderImpl.kt

```diff
@@ -3,19 +3,19 @@ package com.example.coronadiagnosticapp.data.providers
 import android.content.Context
 import javax.inject.Inject
 
-const val USER_TOKEN = "USER_TOKEN"
-const val USER_NAME = "USER_NAME"
-const val IS_FIRST_TIME = "IS_FIRST_TIME"
-
 class SharedProviderImpl @Inject constructor(context: Context) : SharedProvider,
     PreferenceProvider(context) {
 
-
-    override fun getToken(): String? {
-        return preferences.getString(USER_TOKEN, null)
-
+    companion object{
+        const val USER_TOKEN = "USER_TOKEN"
+        const val USER_NAME = "USER_NAME"
+        const val IS_FIRST_TIME = "IS_FIRST_TIME"
+        const val DID_SET_NOTIF_TIME = "DID_SET_NOTIF_TIME"
```

Comment:
It's best practice to start boolean flags with is.
In this case, it could be something like:
IS_NOTIFICATION_TIME_SET

Reply:
ok. it's apple's convention.